src/components/GeoSearchControls/GeoSearchControls.tsx

```diff
@@ -55,6 +55,7 @@ const GeoSearchControls = ({
               >
                 <Template
                   {...templateProps}
+                  $$widgetType="ais.geoSearch"
```

Comment:
instead of this exception, is there some way we could make it work as expected? maybe with a placeholder element we preact.render into?

Reply:
You mean make `html` work in `geoSearch` right now instead of waiting, right?

Reply:
yes, it seems a little odd to have this single exception, without externally clear reason

Reply:
This was mostly because of time constraints. It isn't meant to last.